common/cmake/devices.cmake

```diff
@@ -997,8 +1005,11 @@ function(ADD_FPGA_TARGET)
           symbiflow-arch-defs_SOURCE_DIR=${symbiflow-arch-defs_SOURCE_DIR}
           OUT_EBLIF=${OUT_EBLIF}
           OUT_SYNTH_V=${OUT_SYNTH_V}
+          OUT_FASM_EXTRA=${OUT_FASM_EXTRA}
+          PART_JSON=${PART_JSON}
+          INPUT_XDC_FILE=${INPUT_XDC_FILE}
           ${ADD_FPGA_TARGET_DEFINES}
-          ${QUIET_CMD} ${YOSYS} -p "${COMPLETE_YOSYS_SCRIPT}" -l ${OUT_EBLIF}.log ${SOURCE_FILES}
+          ${QUIET_CMD} ${YOSYS} -p "plugin -i xdc" -p "plugin -i fasm" -p "${COMPLETE_YOSYS_SCRIPT}" -l ${OUT_EBLIF}.log ${SOURCE_FILES}
```

Comment:
Why not just modify "$COMPLETE_YOSYS_SCRIPT"?

Reply:
E.g. https://github.com/SymbiFlow/symbiflow-arch-defs/blob/master/xc7/yosys/synth.tcl

Reply:
Done